app/helpers/hosts_helper.rb

```diff
@@ -24,6 +24,13 @@ def nic_info_js(compute_resource)
     javascript_include_tag("compute_resources/#{compute_resource.provider.downcase}/nic_info.js")
   end
 
+  def value_hash_cache(host)
```

Comment:
for some reason, we call the classification for every class parameter, thus, cache it.